src/cascadia/TerminalApp/GlobalAppSettings.h

```diff
@@ -58,6 +58,17 @@ class TerminalApp::GlobalAppSettings final
     bool GetCopyOnSelect() const noexcept;
     void SetCopyOnSelect(const bool copyOnSelect) noexcept;
 
+    int32_t GetInitialX() const noexcept;
+    void SetInitialX(const int32_t initialX) noexcept;
+    bool GetIsInitialXSet() const noexcept;

```

Comment:
Can we change `GetIsInitial*Set()` to `IsInitial*Set()`? It'll read better when it gets used.

Reply:
Also change `_initialX` and `_initialY` from `int32_t` to `std::optional<int32_t>`. Maybe then we wouldn't even need these two functions.

Reply:
Changed